src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/PSUserAgent.cs

```diff
@@ -118,19 +120,36 @@ internal static string App
             }
         }
 
-        internal static string Platform
+        internal static string PlatformName
         {
             get
             {
-                return ("Windows NT");
+                if (Platform.IsWindows)
+                {
+                    return "Windows NT";
+                }
+                else if (Platform.IsMacOS)
+                {
+                    return "Macintosh";
+                }
+                else if (Platform.IsLinux)
+                {
+                    return "Linux";
+                }
+                else
+                {
+                    // unknown/unsupported platform
+                    Debug.Assert(true, "Unable to determine Operating System Platform");
```

Comment:
I think this should be `Debug.Assert(false, "...")`. And it's recommended to use `Diagnostics.Assert` which is from the `System.Management.Automation` namespace. See [the code here](https://github.com/PowerShell/PowerShell/blob/master/src/Microsoft.PowerShell.Commands.Utility/commands/utility/compare-object.cs#L154) as an example.

Reply:
@daxian-dbw I have adressed your feedback.